zoo-css-challenge/style.css

```
@@ -64,8 +64,13 @@ button {
   background-color: #372772;
   border: 2px solid black;
   color: #fff;
+  font-size: medium;
+  
+}
+button:hover {
```

```suggestion
button:focus,
button:hover {
```
We should also add `:focus` so that when users use the `Tab` key to move to the button, it also changes colour for them